tools/Google.Cloud.Tools.ReleaseManager/CheckVersionCompatibilityCommand.cs

```diff
@@ -142,15 +142,16 @@ private static Level CheckCompatibility(ApiMetadata api, StructuredVersion versi
                 Console.WriteLine($"Returning 'identical' as the change level; please check carefully before release.");
                 return Level.Identical;
             }
-            string[] candidateTfms = { "netstandard2.1", "netstandard2.0" };
+            // Google.Cloud.Diagnostics.AspNetCore3 targets .NET Core 3.1
+            string[] candidateTfms = { "netstandard2.1", "netstandard2.0", "netcoreapp3.1" };
```

Comment:
@jskeet  Can we remove netstandard2.1 since we've re-released everything?

Reply:
I'd prefer to keep it - aside from anything else, that's still what we target for Spanner at the moment.

Reply:
Yep makes sense. And I had forgotten about Spanner.